src/plugins/plot/chart/MctChart.vue

```diff
@@ -531,8 +542,17 @@ export default {
 
             return true;
         },
+        redrawIfNotAlreadyHandled(attribute, value, oldValue) {
+            if (Object.keys(HANDLED_ATTRIBUTES).includes(attribute) && oldValue) {
+                return;
+            }
+
+            console.warn('Unhandled change:', attribute);
```

Comment:
This is producing a ton of warnings for me:

![image](https://user-images.githubusercontent.com/9259993/216400067-d79a208d-140e-4fb7-8024-bda882422a3e.png)

Reply:
NICE! This is what I wanted to see...

Reply:
Handled the 'stats' attribute now.